### DISCUSSION TITLE
I need to download non-svg images (.png) to use with pins

### DISCUSSION THREAD

**Discussion by davefxy** [Mapsui/Mapsui#1681] [Q&A]
I need to download non-svg images (.png) to use with pins. Will I need to convert these to svg images to use them as the Pin Svg parameter. If so, what APIs are available to perform the conversion?

**Comment by charlenni** [comment]
I don't know if any tool, that could convert PNG files (raster files) into SVG files (vector files). The other way would be no problem.

Did you looked at Pin.Icon? I assume, that this is what you want.

**Comment by davefxy** [comment]
How Do I build the mapsui directory so I can run the maui mapview sample with mapsui. I have already created the mapsui maui mapview sample using the mapsui nuget. That works fine. I wanted to step through mapsui code. Sent from Mail for Windows From: charlenniSent: Sunday, July 24, 2022 12:57 AMTo: Mapsui/MapsuiCc: Davefxy; AuthorSubject: Re: [Mapsui/Mapsui] I need to download non-svg images (.png) to use with pins (Discussion #1681) I don't know if any tool, that could convert PNG files (raster files) into SVG files (vector files). The other way would be no problem.Did you looked at Pin.Icon? I assume, that this is what you want.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by davefxy** [comment]
Ignore the previous email about building the mapsui maui mapview sample. I read the error lines which indicated I needed to add the subprojects since I was using Visual Studio to build the project. Then it built just fine. What do developers on this mapsui project do to build and debug the code if they use the existing project files? Docker? Which I think I have used once. Is this something I need to learn how to use for builds? Sent from Mail for Windows From: charlenniSent: Sunday, July 24, 2022 12:57 AMTo: Mapsui/MapsuiCc: Davefxy; AuthorSubject: Re: [Mapsui/Mapsui] I need to download non-svg images (.png) to use with pins (Discussion #1681) I don't know if any tool, that could convert PNG files (raster files) into SVG files (vector files). The other way would be no problem.Did you looked at Pin.Icon? I assume, that this is what you want.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
